erpnext/accounts/doctype/pricing_rule/utils.py

```diff
@@ -30,13 +30,19 @@ class MultiplePricingRuleConflict(frappe.ValidationError): pass
 }
 
 def get_pricing_rules(args, doc=None):
-	pricing_rules = []
-	values =  {}
+	pricing_rules_all = []
+	values = {}
 
 	for apply_on in ['Item Code', 'Item Group', 'Brand']:
-		pricing_rules.extend(_get_pricing_rules(apply_on, args, values))
-		if pricing_rules and not apply_multiple_pricing_rules(pricing_rules):
-			break
+		pricing_rules_all.extend(_get_pricing_rules(apply_on, args, values))
+
+	# removing duplicate pricing rule
+	pricing_rules_title = []
+	pricing_rules = []
+	for p in pricing_rules_all:
+		if p['title'] not in pricing_rules_title:
```

Comment:
use "name"(primary key) instead of "title"

Reply:
I've change `title` to `name`

https://github.com/frappe/erpnext/pull/22328/commits/303639db764672ba9347475419c347aad6d06e02